tfx/components/trainer/executor_test.py

```diff
@@ -186,6 +186,19 @@ def testDoWithHyperParameters(self):
     self._verify_model_exports()
     self._verify_model_run_exports()
 
+  def testDoWithCustomSplits(self):
```

Comment:
Test is for custom splits but it's just using the default 'train' and 'eval' splits. So test results would be no different if those were omitted. Better to use actual custom splits.

Reply:
To use custom splits, we need to add a custom split folder to the iris data. Currently, the tfx/components/testdata/iris/data folder only contains 'eval' and 'train'. Should I add another folder under testdata for this test?

Reply:
@1025KB Do you have a preference?

Either options sound good to me.